code/__DEFINES/reagents.dm

```diff
@@ -2,6 +2,25 @@
 #define LIQUID			2
 #define GAS				3
 
+/// Moles per reagent unit. Used to convert between reagent units and gases.
+#define REAGENT_MOLE_DENSITY	10
+/// The boiling point of water.
+#define BP_WATER			(T0C+100)
+/// The boiling point of nitrogen.
+#define BP_NITROGEN			(300+10)
+/// The boiling point of oxygen.
+#define BP_OXYGEN			(300+10)
```

Comment:
Because if I make it the actual boiling point of nitrogen/oxygen it will boil the instant it comes out of the chem dispenser and I don't want to drive chemists up a wall. This can be changed once the primary way of acquiring oxygen/nitrogen is not dispensing it in liquid form into an open-topped container.

Reply:
No like why is it number + number rather then either 310 or TOC + 10

Reply:
My brain was still running on the same logic that made water's boiling point T0C + 100. I'll fix it when I get back to a computer.

Reply:
Should be done now.